build/common.go

```diff
@@ -192,6 +196,45 @@ func BuildAll() { //revive:disable-line
 	mg.Deps(b.Linux, b.Windows, b.Darwin, b.DarwinARM64, b.LinuxARM64, b.LinuxARM)
 }
 
+//go:embed tmpl/*
+var tmpl embed.FS
+
+// ensureWatchConfig creates a default .bra.toml file in the current directory if it doesn't exist.
+func ensureWatchConfig() error {
+	exists, err := utils.Exists(".bra.toml")
+	if err != nil {
+		return err
+	}
+
+	if exists {
+		return nil
+	}
+
+	fmt.Println("No .bra.toml file found. Creating one...")
+	config, err := tmpl.ReadFile("tmpl/bra.toml")
+	if err != nil {
+		return err
+	}
+
+	return os.WriteFile(".bra.toml", config, 0600)
+}
+
+// Watch rebuilds the plugin backend when files change.
+func Watch() error {
+	if err := ensureWatchConfig(); err != nil {
+		return err
+	}
+
+	// this is needed to run `bra run` programmatically
+	app := cli.NewApp()
+	app.Name = "bra"
+	app.Usage = ""
+	app.Action = func(c *cli.Context) error {
+		return bra.Run.Run(c)
+	}
+	return app.Run(os.Args)
```

Comment:
Would be good to verify this works on Windows, maybe you can ask someone when this has been merged?